src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -107,7 +164,10 @@ public UnorderedCallAssertion MustHaveHappened(Repeated repeatConstraint)
         }
 
         public class ReturnValueConfiguration<TMember>
-            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>, ICallCollectionAndCallMatcherAccessor
+            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>,
+              IAfterCallSpecifiedWithOutAndRefParametersConfiguration<IReturnValueConfiguration<TMember>>,

```

Comment:
Oh! A dig about the delay in the review. Nasty, M. Levesque.

(I'm just teasing.)